consent/handler.go

```diff
@@ -250,13 +251,18 @@ func (h *Handler) DeleteLoginSession(w http.ResponseWriter, r *http.Request, ps
 //     Responses:
 //       200: loginRequest
 //       401: genericError
+//       409: genericError
 //       500: genericError
 func (h *Handler) GetLoginRequest(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	request, err := h.M.GetAuthenticationRequest(r.Context(), ps.ByName("challenge"))
 	if err != nil {
 		h.H.WriteError(w, r, err)
 		return
 	}
+	if request.WasHandled {
+		h.H.WriteError(w, r, pkg.ErrConflict)
```

Comment:
I think it would be better to have a descriptive error here like the one from the manager: `fosite.ErrInvalidRequest.WithDebug("Authentication verifier has been used already")`.

Reply:
Ok. I'll add it